### ISSUE TITLE
Составить дополнительные информационные блоки

### ISSUE THREAD

**Issue by neonila-kunyk** [neonila-kunyk/homepage#7]
Можно сделать резюме менее формальным, добавив дополнительные блоки, которые содержат полезную информацию, но подают её в "развлекательной" форме. Например, что нравится и не нравится, сильные и слабые стороны, перечень навыков или хобби, инфографика или же интерактивные модули, и т.п. Стоит написать контент такого блока раньше, чем начинать верстку, потому что его размеры будут влиять на размещение элементов на странице.

**Comment by neonilakunyk** [comment]
**SMILEY LORE :-)**
Yes, I am the inventor of the sideways “smiley face” (sometimes called an “emoticon”) that is commonly used in E-mail, chat, and newsgroup posts. The historic day was September 19, 1982.
We had local email and had just gained the possibility of email to a few other universities on the primitive networks. But this was text only, using only the U.S. English alphabet, so we didn’t have much to work with.
The unique character sequences caught on quickly at Carnegie Mellon and proceeded to infiltrate computer network message boards at other universities and research labs around the world. Within a few months, different “smileys” emerged, and before long, there were dozens of them.
I didn’t keep the original post where I made the emoticon suggestion, largely because I didn’t think it was that big of a deal. But when the smiley face turned into a full-blown phenomenon, I knew I had to seriously look for it.
This was 10 minutes of my life, just part of a silly online discussion more than 35 years ago. So, it’s kind of a strange thing to be famous for, but it’s fun to be a little bit famous for something. I’ve come to accept that no matter what I achieve in my career as an AI researcher, the first sentence of my obituary will talk about the smiley face invention. If I’ve prevented some online misunderstandings along the way, and if people have had fun with these things, then I guess that 10 minutes was well spent.

The emoticon Fahlman posted to the local bulletin board looked like this:
19-Sep-82 11:44
Scott E Fahlman :-)
From: Scott E Fahlman
I propose that the following character sequence for joke markers: :-) Read it sideways.Actually, it is probably more economical to mark things that are NOT jokes, given current trends. For this, use :-(

**CHARACTER**
Motivation. Critical thinking. Purposefulness. Confidence. Curiosity. Optimism. Flexibility of the mind. Willingness to take risks. Independence of thinking. The ability to perceive new ideas. Fantasy. Perseverance. Concentration of attention. Figurative thinking. Sense of humor. Discipline.

**LIKES**
Quotes. Digital photography. Cooking. Writing. Logics. Numbers. Technologies. Accuracy. Books. Sudoku. Stars. Intellect. Laws. Speed. Greens. Porcelain. Architecture. History. Ancient castles. Sausages. Road. Sea. Islands. Nature. Travels. Blue colour.

**DISLIKES**
Gambling. Mouse. Reptiles. Chaos. Height. Pointless chatter. Incompetence. Arrogance. Irresponsibility. Envy. Lies. Cruelty. Impudence. Rudeness. Wars. Wind. Mud. Potatoes. The smell of cigarettes.

**FAVORITE QUOTES**
·      “The secret to doing good research is always to be a little underemployed. You waste years by not being able to waste hours.”
Amos Tversky
·      “If a machine is expected to be infallible, it cannot also be intelligent.”
Alan Turing, 1947
·      “Nothing is less productive than to make more efficient what should not be done at all.”
Peter Drucker
·      “The best ideas come as jokes. Make your thinking as funny as possible.”
David M. Ogilvy
·      “First things first, but not necessarily in that order.”
Dr. Who
·      “A manager should be like the sweeper in curling: The sweeper runs ahead of the stone and sweeps away debris from the path of the stone so that the progress of the stone will be smooth and undisturbed.”
Richard P. Gabriel
